### PR TITLE
docs(Table column management demo): Added demo

### LINKED ISSUE #3898: PF4: Table column management demo
**Describe the issue. What is the expected and unexpected behavior?**
Add Table column management demo

related core issue https://github.com/patternfly/patternfly-next/issues/2626

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Added table column management demo, which illustrates how you can can remove/add columns from tables.

Users can click checkboxes in the modal to edit which columns are displayed.

Screen Shot 2020-03-20 at 11 52 32 AM

Screen Shot 2020-03-20 at 11 52 40 AM

Fixes #3898.

### CODE REVIEW

**Comment by rebeccaalpert** [comment]
Just pushed to rebase package.json; didn't change anything else. (Thanks for the heads up @redallen.)

File: packages/react-core/src/demos/TableColumnManagementDemo.md [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+        case 'table-column-management-check-action-check2':
+          return 'Branches';
+        case 'table-column-management-check-action-check3':
+          return 'Pull requests';
+        case 'table-column-management-check-action-check4':
+          return 'Workspaces';
+        case 'table-column-management-check-action-check5':
+          return 'Last commit';
+      }
+    };
```
Comment (dlabrecq) [inline packages/react-core/src/demos/TableColumnManagementDemo.md]: I find the name conversions confusing. In Cost Management, we just provide the names we intend to use without mappings.

Can we not provide the table checkbox with the name "check1", "check2", etc., for the purpose of this example? (My first preference)  Or, change the state names to use the matching name?

Comment (rebeccaalpert) [reply packages/react-core/src/demos/TableColumnManagementDemo.md]: I was copying the id scheme Core was using. Just got @mcoker's blessing to diverge.

**Comment by rebeccaalpert** [comment]
Updated text and checkbox names; rebased.

**Comment by rebeccaalpert** [comment]
Rebased again.

**Pull request merged**